rbac/management/audit_log/model.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Model for audit logging."""
+from django.db import models
+from django.utils import timezone
+
+from api.models import Tenant, TenantAwareModel
+
+
+class AuditLog(TenantAwareModel):
+    """An audit log."""
+
+    GROUP = "group"
+    ROLE = "role"
+    USER = "user"
+    PERMISSION = "permission"
+    RESOURCE_CHOICES = (
+        (GROUP, "Group"),
+        (ROLE, "Role"),
+        (USER, "User"),
+        (PERMISSION, "Permission"),
+    )
+
+    DELETE = "delete"
+    ADD = "add"
+    EDIT = "edit"
+    CREATE = "create"
+    REMOVE = "remove"
+    ACTION_CHOICES = (
+        (DELETE, "Delete"),
+        (ADD, "Add"),
+        (EDIT, "Edit"),
+        (CREATE, "Create"),
+        (REMOVE, "Remove"),
+    )
+
+    created_at = models.DateTimeField(default=timezone.now)
+    requester = models.TextField(max_length=255, null=False)
+    description = models.TextField(max_length=255, null=False)
+    resource = models.CharField(max_length=32, choices=RESOURCE_CHOICES)
+    action = models.CharField(max_length=32, choices=ACTION_CHOICES)
+    org_id = models.TextField(max_length=255, null=True)
+    tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE, null=True)
```

Comment:
This is not needed here as it is defined in `TenantAwareModel`

Reply:
When I was running the migrations for the table, I kept on receiving a error that the field can't be empty, which is why I had that there within the model

Reply:
Okay, I re-tested with the changes that I made and it seems to be working properly, so I don't know what I did beforehand